assets/src/edit-story/components/library/libraryTabs.js

```diff
@@ -35,7 +36,14 @@ function LibraryTabs() {
     data: { tabs },
   } = useLibrary();
   const { isRTL } = useConfig();
-  const panes = useMemo(() => getPanes(tabs), [tabs]);
+  const { animationTab } = useFeatures();
```

Comment:
Small nit: most flags are in the form of a verb like `showsAnimationTab` or `hasAnimationTab`. Without that I could think this might contain the JSX for the tab.